ImportTool/RequestGenerator.cs

```diff
@@ -0,0 +1,120 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace ImportTool
+{
+    public static class RequestGenerator
+    {
+        public static async Task<string> GenerateImportRequest(string connectionString, string prefix, int maxFileNumber)
```

Comment:
why we have this limit?

Reply:
It is not a limit, user can choose how much files they want to set as input